sdk-wasm/src/lib.rs

```diff
@@ -2,7 +2,25 @@
 #![cfg(target_arch = "wasm32")]
 #[deprecated(since = "2.2.0", note = "Use solana_instruction::wasm instead.")]
 pub use solana_instruction::wasm as instructions;
-use wasm_bindgen::prelude::*;
+use {::log::Level, wasm_bindgen::prelude::*};
+pub use {
+    solana_program::*,
+    solana_sdk::{
+        // These imports exist in both solana_sdk and solana_program, so we use
+        // direct imports to suppress ambiguous re-export warnings.
+        declare_deprecated_id,
+        declare_id,
+        entrypoint,
+        entrypoint_deprecated,
+        example_mocks,
+        feature,
+        hash,
+        program_utils,
+        pubkey,
+        *,
+    },
+};
```

Comment:
This is probably all that we need from the new crate, everything else should live in its original crate.